content/id/docs/concepts/extend-kubernetes/extend-cluster.md

```diff
@@ -0,0 +1,188 @@
+
+*API kebijakan bawaan*, seperti [ResourceQuota](/docs/concepts/policy/resource-quotas/), [PodSecurityPolicies](/docs/concepts/policy/pod-security-policy/), [NetworkPolicy](/docs/concepts/services-networking/network-policies/) dan Role-based Access Control ([RBAC](/docs/reference/access-authn-authz/rbac/)), adalah API bawaan Kubernetes.
+API biasanya digunakan oleh layanan Kubernetes yang dihosting dan diatur oleh instalasi Kubernetes. Mereka bersifat deklaratif dan menggunakan konvensi yang sama dengan sumber lain Kubernetes seperti
+pods, jadi konfigurasi kluster baru dapat diulang-ulang dan dapat diatur dengan cara yang sama dengan aplikasi. 
+Dan, ketika mereka stabil, mereka menyukai [kebijakan pendukung yang ditetapkan](/docs/reference/deprecation-policy/) seperti API Kubernetes lainnya. Oleh karena itu, mereka lebih disukai daripada
+*konfigurasi file* dan *bendera* yang sesuai.
+
+## Ekstensi
+
+Ekstensi adalah komponen perangkat lunak yang luas dan terintegrasi secara dalam dengan Kubernetes.
+Mereka mengadaptasi Kubernetes untuk mendukung perangkat keras tipe baru dan jenis baru.
+
+Kebanyakan admin kluster akan menggunakan instansi Kubernetes yang terdistribusi atau yang dihosting.
+Sehingga hasilnya adalah, kebanyakan pengguna Kubernetes akan membutuhkan instalasi ekstensi dan beberapa
+akan butuh untuk membuat baru.
+
+## Pola Ekstensi
+
+Kubernetes didesain untuk dapat diotomasi dengan menuliskan program klien. Program apapun
+yang membaca dan/atau menulis ke API Kubernetes dapat menyediakan otomasi yang berguna. 
+
+*Otomasi* dapat berjalan di dalam kluster atau di luar kluster. Dengan mengikuti panduan
+di dalam dokumen ini, Anda dapat menulis otomasi yang sangat tersedia dan kuat.
+Otomasi pada umumnya dapat bekerja dengan berbagai macam kluster Kubernetes, termasuk
+kluster yang terhosting dan instalasi yang diatur.
+
+Ada pola spesifik tertentu untuk menulis program klien yang bekerja baik dengan Kubernetes
+yang disebut pola *Controller*. *Controller* biasanya membaca objek milik `.spec`, kemungkinan
+melakukan sesuatu, dan kemudian memperbarui objek milik `.status`.
+
+*Controller* adalah klien dari Kubernetes. Ketika Kubernetes adalah klien dan memanggil layanan
+jarak jauh, ini disebut *Webhook*. Layanan jarak jauhnya disebut *Webhook Backend*. Seperti *controller*, Webhooks
+memang menambah poin kegagalan.
+
+Di dalam model *Webhook*, Kubernetes membuat sebuah *network request* kepada sebuah layanan jarak jauh.
+
+Di dalam model *Binary Plugin*, Kubernetes mengeksekusi file binari (program).
+Plugin binari digunakan oleh kubelet (seperti [Plugin Volume Flex](https://github.com/kubernetes/community/blob/master/contributors/devel/flexvolume.md)
+dan [Plugin Jaringan](/docs/concepts/cluster-administration/network-plugins/)) dan oleh kubectl.
+
+Berikut ini adalah diagram yang menunjukkan bagaimana titik ekstensi berinteraksi dengan bidang kontrol Kubernetes.
```

Comment:
```suggestion
Berikut ini adalah diagram yang menunjukkan bagaimana titik-titik ekstensi berinteraksi dengan _control plane_ Kubernetes.
```

Let's use italicized english word to avoid confusion.

Reply:
done